tests/test_layout.py

```diff
@@ -39,3 +39,72 @@ def test_that_product_older_versions_are_ordered_correctly(self, mozwebqa):
         csp = CrashStatsHomePage(mozwebqa)
 
         Assert.is_sorted_descending(csp.header.other_versions)
+
+
+class TestSuperSearchLayout:
```

Comment:
I didn't know that we could add a second class in a test file or creating another test file. Just wanted to ask